projects/module-2/whats-cookin-part-two-agile.md

```diff
@@ -0,0 +1,206 @@
+
+Being able to do a DELETE request is not a requirement for this project. So to delete a user’s favorite recipes, you can reset the data on the local server by restarting the local server on your machine. Do not worry about “un-favoriting” a recipe via a DELETE request; restart the API server instead.
+
+### Fetch
+
+You will no longer receive your data from the deployed API, but rather implementing the fetch API for accessing the data from a [local server](https://github.com/turingschool-examples/whats-cookin-api).
+
+Your GET requests should not change very much, but you will need to clone down the repo for the local server and run it in a separate tab in your terminal each time you run your client (app). Below are the endpoints’ setup for this project.
+
+### Why this change?
+
+Everyone was using the same exact endpoints for Part One. This is typically how things work, but can cause some headache when implementing types of requests other than GET. In this part of the project, you will start making POST requests. Creating a local server instead of using the hosted (deployed) one, gives everyone their own version of the APIs and decreases an overload of new data getting sent.
+
+<section class="note">
+### Remember:
+
+The expectation for Mod 2 is that you will avoid using `async/await`. We know `async/await` is tempting, but it is important that you are able to work with the approaches that pre-date the introduction of `async/await`. Consider doing some research on [Promise.all()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all).
+
+Also, for POST requests, your request will need a header of `Content-Type: application/json`.
+</section>
+
+### Endpoints (GET and POST)
+
+| Description | URL | Method | Required Properties for Request | Sample Successful Response |
+| --- | --- | --- | --- | --- |
+| Get all users | http://localhost:3001/api/v1/users | GET | none | An array containing all users |
+| Get all ingredients | http://localhost:3001/api/v1/ingredients | GET | none | An array containing all ingredients |
+| Get all recipes | http://localhost:3001/api/v1/recipes | GET | none | An array containing all recipes |
+| Add recipe to cook for a user | http://localhost:3001/api/v1/usersRecipes | POST | { userID: `<number>`, recipeID: `<number>` } | {message: "Recipe # `<recipeID>` was added for User # `<userID>`" } |
+
+## Error Handling
+
+Make proper error handling for your users to ensure they get data and submit their POST requests successfully. An example is handling the case where they submit their data and an error message returns from the request. How would you show helpful feedback to a user?
+
+---
+
+## Iteration 5 - Differentiation Tracks
+
+Each team will get **one** of these tracks to work through.
```

Comment:
Maybe instead of "get" the language could be "be assigned"? If the PMs are assigning these tracks.